Improve `README.md` (#4270)
boa-dev/boa: README.md

    </a>
</p>

> `Boa` is an experimental JavaScript lexer, parser and interpreter written in Rust 🦀, it has support for **more** that `90%` of the latest ECMAScript specification. We continuously improve the conformance to keep up with the ever-evolving standard.

Reviewer: ```suggestion
`Boa` is an experimental JavaScript lexer, parser and interpreter written in Rust 🦀, it has support for **more** than `90%` of the latest ECMAScript specification. We continuously improve the conformance to keep up with the ever-evolving standard.
```

> Congrats! You've executed your first `JavaScript` using `Boa`!

Reviewer 2: ```suggestion
Congrats! You've executed your first JavaScript code using Boa!
```

## 🔎 Documentation

> For more information on `Boa`'s API. Feel free to check out our documentation.

Reviewer 2: ```suggestion
For more information on Boa's API. Feel free to check out our documentation.
```

[**API Documentation**](https://docs.rs/boa_engine/latest/boa_engine/)

## 🏅 Conformance

To know how much of the _ECMAScript_ specification does Boa cover, you can check out results
running the _ECMASCript Test262_ test suite [here](https://boajs.dev/conformance).

## 🪚 Contributing

Please, check the [CONTRIBUTING.md](CONTRIBUTING.md) file to know how to
contribute in the project. You will need Rust installed and an editor. We have
some configurations ready for VSCode.

### 🐛 Debugging

Check [debugging.md](./docs/debugging.md) for more info on debugging.

> ### 🕸 Web Assembly

Reviewer: question: Why a spiderweb? Is that used elsewhere for web assembly?

Author: > question: Why a spiderweb? Is that used elsewhere for web assembly?

I couldn’t come up with a better option... It kind of fits since it’s related to WebAssembly. :sweat_smile:
I haven't seen it anywhere, most repos that I've seen that use emojis for sections are a lot of NeoVim packages.
